Automattic/wp-calypso
# Plans 2023: plans data-store foundation

## Proposed Changes

It connects the 2023 plans grid to the plans data-store and makes it a dependency for loading the grids i.e. the grids will not render until the plans' data are fetched and loaded in the store.

It does not introduce any selector or other functionality. We should be able to start tuning and using now. The setup will improve as more things from the plans' code are refactored and deprecated.

## Testing Instructions



* Go to `/plans/[SITE]` and ensure the grids are rendered
* Same for `/start/plans`

## Pre-merge Checklist
